clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchResponse.java

```diff
@@ -19,6 +19,7 @@
 import java.util.Map.Entry;
```

Comment:
As I follow-up, I would like to clean this class. There are way too many ways to construct this object and the logic is pretty complicated. It is the same for the request. Let's do this separately.

Reply:
sounds like a good plan. Thanks!